pcsx2/x86/ix86-32/iR5900-32.cpp

```diff
@@ -710,7 +710,7 @@ static void recResetRaw()
 
 	maxrecmem = 0;
 
-	memset(recConstBuf, 0, RECCONSTBUF_SIZE * sizeof(recConstBuf));
+	memset(recConstBuf, 0, RECCONSTBUF_SIZE * sizeof(*recConstBuf));
```

Comment:
If I trust coverity report, there is no impact. The type is the same size as the pointer. However it will crash on 64 bits.